Doc/using/windows.rst

```diff
@@ -1109,7 +1111,7 @@ For extension modules, consult :ref:`building-on-windows`.
       MinGW gcc under Windows" or "Installing Python extension with distutils
       and without Microsoft Visual C++" by Sébastien Sauvage, 2003
 
-   `MingW -- Python extensions <http://oldwiki.mingw.org/index.php/Python%20extensions>`_
+   `MingW -- Python extensions <http://www.mingw.org/wiki/FAQ#toc14>`_
       by Trent Apted et al, 2007
```

Comment:
Should we remove this credit since the document has changed (from https://web.archive.org/web/20160412131224/http://oldwiki.mingw.org/index.php/Python%20extensions) and now an FAQ?

Reply:
Yes.